framework/wazuh/core/cluster/dapi/dapi.py

```diff
@@ -336,11 +336,11 @@ def get_client(self) -> c_common.Handler:
 
         :return: client. Maybe an instance of LocalClient, WorkerHandler or MasterHandler
         """
-        if self.node == local_client:
+        if isinstance(self.node, local_client.LocalClient):
+            client = self.node
+        else:
```

Comment:
This actually fixes the problem, but the implications of the change may be bigger than it seems, perhaps too big to analyze at this time. 

In this use case (which is a bit complicated), the `execute` method of the MasterHandler class was run. Now, it would be run in the LocalServer. I think it will be less prone to unexpected failures if you modify the header below to add a default value for the `wait_for_complete` parameter:
https://github.com/wazuh/wazuh/blob/813595cf58d753c1066c3e7c2018dbb4708df088/framework/wazuh/core/cluster/master.py#L304-L305